cmd/kube-proxy/app/server_test.go

```diff
@@ -625,3 +625,76 @@ func TestAddressFromDeprecatedFlags(t *testing.T) {
 
 	}
 }
+
+func TestFlagOverwrites(t *testing.T) {
```

Comment:
Don't need to be on this PR but we can extend the testing for platform and instances specific ones.

Reply:
Thanks @knabben, good point! I'll add coverage once reviewers reach a consensus on this overwriting behavior.

Reply:
Thanks @knabben, good point! I'll add coverage once reviewers reach a consensus on this overwriting behavior. The test for now is just to demo flags will take precedence on proxy component configs.